filebeat/input/filestream/input.go

```diff
@@ -235,6 +240,9 @@ func (inp *filestream) openFile(path string, offset int64) (*os.File, error) {
 		return nil, fmt.Errorf("failed opening %s: %s", path, err)
 	}
 
+	if fi.Size() < offset {
+		offset = 0
```

Comment:
This means we have just detected another truncate event. The harvester already logs events on the file at info level. We should also emit a log message here if we detect that the file has been truncated.

Do we need the prospector to log as well?

Reply:
Prospector already logs on debug level that a file was truncated. Should I increase the logging level to info?
I did not add a log here because the prospector already emits a logline but it might make our jobs easier, in the long run, to see everytime Filebeat detects truncation.

Reply:
Yes, harvester/input states have historically been logged at the info level. I would like to keep it that way.